### ISSUE TITLE
Error  "DB instance does not have specified type Public connection" but a public connection actually was created 

### ISSUE THREAD

**Issue by pierrchen** [alibaba/terraform-provider#410]
Hi,

TF showed following error when trying to allocation a public connection for a rds.

```
* alicloud_db_connection.connection: AllocateInstancePublicConnection got an error: &errors.errorString{s:"WaitForDBConnection got error: &common.Error{ErrorResponse:common.ErrorResponse{Response:common.Response{RequestId:\"\"}, HostId:\"\", Code:\"Instance.Notfound\", Message:\"DB instance rm-j0b17zw6363ydh500 does not have specified type Public connection.\"}, StatusCode:-1}"}
```
However, despite of the error, a public connection has actually being created: 

```
 ./out/aliyun rds DescribeDBInstanceNetInfo --DBInstanceId rm-j0b17zw6363ydh500
{"DBInstanceNetInfos":{"DBInstanceNetInfo":[{"SecurityIPGroups":{"securityIPGroup":[]},"Port":"3306","ConnectionString":"rm-j0b17zw6363ydh500.mysql.australia.rds.aliyuncs.com","ConnectionStringType":"Normal","VPCId":"vpc-p0ww48a41x0wthp6603j2","IPAddress":"172.16.6.96","IPType":"Private","Upgradeable":"Disabled","DBInstanceWeights":{"DBInstanceWeight":[]},"VSwitchId":"vsw-p0wpqek0r5o2k7v9jxujl"},{"SecurityIPGroups":{"securityIPGroup":[]},"Port":"3306","ConnectionString":"xcxcxcx.mysql.australia.rds.aliyuncs.com","ConnectionStringType":"Normal","VPCId":"","IPAddress":"47.91.37.129","IPType":"Public","Upgradeable":"Disabled","DBInstanceWeights":{"DBInstanceWeight":[]}}]},"RequestId":"06BCBB95-5FA8-42E2-94FD-B640EE09CD2F","InstanceNetworkType":"VPC"}
```
Is it error message a bug and how should I interpret it?  @xiaozhu36

**Comment by xiaozhu36** [comment]
Hi @pierrchen Which is your terraform alicloud provider version?

**Comment by pierrchen** [comment]
Hi,
@xiaozhu36  I'm using v1.8.0

Terraform v0.11.3
+ provider.alicloud v1.8.0

**Comment by xiaozhu36** [comment]
Hi @pierrchen Did you try again? It may be a random bug.

**Comment by pierrchen** [comment]
@xiaozhu36  I just gave it another try but hit same error.

I'm not sure but could it be somehow the deprecated db_instance_net_type filed was expected by the backend? 

[1] https://www.terraform.io/docs/providers/alicloud/r/db_instance.html#db_instance_net_type

Error: Error applying plan:
1 error(s) occurred:

* alicloud_db_connection.connection: 1 error(s) occurred:

* alicloud_db_connection.connection: AllocateInstancePublicConnection got an error: &errors.errorString{s:"WaitForDBConnection got error: &common.Error{ErrorResponse:common.ErrorResponse{Response:common.Response{RequestId:\"\"}, HostId:\"\", Code:\"Instance.Notfound\", Message:\"DB instance rm-j0btke17kti8j96kk does not have specified type Public connection.\"}, StatusCode:-1}"}

**Comment by xiaozhu36** [comment]
Hi @pierrchen New version 1.8.1 has been released and please use it to try again.

**Comment by pierrchen** [comment]
@xiaozhu36  The issue was fixed in 1.8.1. Awesome. Thank you!
